build.gradle.kts

```diff
@@ -154,8 +152,9 @@ subprojects {
         }
     }
 
-    if (!this.name.contains("sample") && !this.name.contains("integration-tests") && this.name != "sentry-test-support" && this.name != "sentry-compose-helper") {
+    if (!this.name.contains("sample") && !this.name.contains("integration-tests") && this.name != "sentry-test-support" && this.name != "sentry-compose-helper" && this.name != "sentry-bom") {
```

Comment:
not needed for sentry-bom anymore?

Reply:
I added this due to `sentry-bom` defining it's own maven publication, causing a duplicate publication error. But actually that's not needed anymore, so I'll remove it from `sentry-bom/build.gradle` instead 👍